app/templates/explore.html

```diff
@@ -11,6 +11,122 @@ <h1>Explore</h1>
+                        <form class="chat-form" action="/chat" method="POST">
+                            <input type="hidden" name="temp" value="{{ ny_temp }}">
+                            <button type="submit" class="btn btn-primary btn-lg px-4 gap-3">Get Your Outfit Suggestion!</button>
+                        </form>
+                    </div>
+                </div>
+
+                <!--San Francisco -->
+                <div class="col-lg-4">
+                    <img class="bd-placeholder-img rounded-circle" width="140" height="140" src="static/San_Francisco.jpeg" alt="San Francisco">
+                    <br> 
+                    <br>
+                    <h2 class="fw-normal">San Francisco</h2>
+                    <p>Discover the charm of the Bay Area!</p>
+                    <!-- Display weather information -->
+                    <div class="mb-3">
+                        {% if sf_temp %}
+                            <p>The current temperature in San Francisco is {{ sf_temp }}°F.</p>
+                        {% else %}
+                            <p>Unable to fetch the current temperature.</p>
+                        {% endif %}
+                    </div>
+                    <div class="d-grid gap-2 d-sm-flex justify-content-sm-center">
+                        <form class="chat-form" action="/chat" method="POST">
+                            <input type="hidden" name="temp" value="{{ sf_temp }}">
+                            <button type="submit" class="btn btn-primary btn-lg px-4 gap-3">Get Your Outfit Suggestion!</button>
+                        </form>
+                    </div>
+                </div>
+
+                <!-- Chicago -->
+                <div class="col-lg-4">
+                    <img class="bd-placeholder-img rounded-circle" width="140" height="140" src="static/Chicago.jpg" alt="Chicago">
+                    <br> 
+                    <br>
+                    <h2 class="fw-normal">Chicago</h2>
+                    <p>The Windy City awaits!</p>
+                    <!-- Display weather information -->
+                    <div class="mb-3">
+                        {% if chicago_temp %}
+                            <p>The current temperature in Chicago is {{ chicago_temp }}°F.</p>
+                        {% else %}
+                            <p>Unable to fetch the current temperature.</p>
+                        {% endif %}
+                    </div>
+                    <div class="d-grid gap-2 d-sm-flex justify-content-sm-center">
+                        <form class="chat-form" action="/chat" method="POST">
+                            <input type="hidden" name="temp" value="{{ chicago_temp }}">
+                            <button type="submit" class="btn btn-primary btn-lg px-4 gap-3">Get Your Outfit Suggestion!</button>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!-- Suggestion Card -->
```

Comment:
something about the suggestion card isn't aligning right--it seems super long and narrow. I think it should look more like it does on the homepage.